app/components/message_draft_component.html.erb

```diff
@@ -10,7 +10,7 @@
           <% if @message.correctly_created? %>
             <div class="gap-2 p-6 border-t-0 border-r-0 border-b border-l-0 border-gray-200">
             <div id="<%= dom_id(@message, :submission) %>" is="turbo-frame" class="flex gap-2">
-              <%= form_with(url: submit_message_draft_path(@message), method: :post, data: { turbo_frame: "_top" }, local: true) do |form| %>
+              <%= form_with(url: submit_message_draft_path(@message), method: :post, data: { turbo_frame: "_top", turbo_confirm: ("Správa ešte nebola podpísaná všetkými podpismi. Naozaj chcete odoslať správu aj bez nich?" if @message.any_objects_with_requested_signature?) }, local: true) do |form| %>
```

Comment:
Predstavme si situaciu, ze mam otvoreny gopro, je tam sprava na podpis, kolega klikne u seba - podpise a ja potom kliknem, ze to chcem odoslat. toto vyhodi spravu, ze tam este nieco treba podpisat ze?

imho jediny sposob ako toto spravit je tu validaciu checkovat ked sa to deje (ten klik)

Reply:
Upravene